modules/data.land/NAMESPACE

```diff
@@ -48,6 +48,8 @@ export(to.TreeCode)
 export(write_ic)
 export(write_veg)
 import(XML)
+import(dplyr)
+import(purrr)
```

Comment:
Things should not be added to the NAMESPACE file by hand. If these two libraries are needed they need to be in the DESCRIPTION file and then you should rerun make document

Reply:
These were added I guess as a result of adding `@import XML dplyr purrr`. I didn't add them manually. But if you think calling import is unnecessary I can take it out.

Reply:
I mean `roxygen` automatically added them to namespace by calling `@import`

Reply:
Yes, if you clean up the `@import` and put the libraries in the DESCRIPTION, then it should be safe to delete these lines.

Reply:
Yeah, we want to avoid importing entire package namespaces, to the point where I would actually recommend _forbidding_ it as part of our style guide. For foreign package function calls, the very strong preference is to call them as `package::function`, and include `package` in the `DESCRIPTION` `imports`. For a small number of edge cases (really, just for operators like `magrittr::%>%`, `rlang::%||%`, `rlang::!!`, etc.), you can use `@importFrom package function`.